app/test/e2e/vocabulary-find-by-ids.spec.js

```diff
@@ -25,7 +25,7 @@ describe('Find vocabularies by IDs', () => {
 
         requester = await getTestServer();
 
-        Resource.remove({}).exec();
+        await Resource.deleteMany();
```

Comment:
I think the `exec()` bit is still needed.

Reply:
Added exec() to deleteMany calls 👍